packages/integration-tests/src/integration/disk-encryption/step-0-disk-encryption.spec.js

```diff
@@ -0,0 +1,107 @@
+import { clusterDetailsPage } from '../../views/clusterDetails';
+import { bareMetalDiscoveryPage } from '../../views/bareMetalDiscovery';
+import { commonActions } from '../../views/common';
+import { tangServerValues } from '../../fixtures/disk-encyrpiton'
+import * as utils from '../../support/utils';
+import {transformBasedOnUIVersion} from "../../support/transformations";
+
+const NEW_CLUSTER_URL = '/clusters/~new';
+
+const getTangServerUrl = (index) => {
+  return cy.get(`#form-input-diskEncryptionTangServers-${index}-url-field`);
+};
+
+const getTangServerThumbprint = (index) => {
+  return cy.get(`#form-input-diskEncryptionTangServers-${index}-thumbprint-field`);
+};
+
+const fillTangServers = (index) => {
+  const TangServerUrl = getTangServerUrl(index);
+  TangServerUrl.type(tangServerValues.Url);
+  TangServerUrl.should('have.value', tangServerValues.Url);
+  const TangServerThumbprint = getTangServerThumbprint(index);
+  TangServerThumbprint.type(tangServerValues.Thumbprint);
+  TangServerThumbprint.should('have.value', tangServerValues.Thumbprint);
+};
+
+const enableDiskEncyrption = () => {
```

Comment:
```suggestion
const enableDiskEncryption = () => {
```

Please look for the `encyrption` word as it's in a few places, including one file name.